src/PlotRecipes/SolidStateDetector.jl

```diff
@@ -12,14 +12,7 @@ end
 
 @recipe function f(contact::Contact)
     linecolor --> contact.id
-    l = contact.name != "" ? contact.name : "Contact $(contact.id)"
-    label --> l
-    contact.geometry
-end
-
-@recipe function f(contact::Contact)
-    linecolor --> contact.id
-    l = contact.name != "" ? contact.name : "Contact $(contact.id)"
+    l = contact.name != "" ? "$(contact.name) (id: $(contact.id))" : "Contact - id: $(contact.id)"
```

Comment:
Is this inconsistency intended (using brackets for no-name contacts and using a hyphen for named contacts?

Reply:
Yes it is. For contacts with names it is supposed to be an additional optional information whereas for no-name contacts it is a non-optional information: `<Name> (id)` <-> `<Type> - id`.

Reply:
Alright!